### PR TITLE
tests: just use full length "uuids" instead of truncating them

### PR DESCRIPTION
**Pull request by phlogistonjohn**
The previous methods were either non-functional when using the fake
uuid generator or a bit more complex than needed for simple unit
test fake-data generation. Units test pass with these full-length ids.

Signed-off-by: John Mulligan <jmulligan@redhat.com>

### CODE REVIEW

File: apps/glusterfs/node_entry_test.go [hunk 1]
```diff
@@ -22,12 +22,11 @@ import (
...
 
 func createSampleNodeEntry() *NodeEntry {
-	g := utils.GenUUID()
 	req := &api.NodeAddRequest{
 		ClusterId: "123",
 		Hostnames: api.HostAddresses{
-			Manage:  []string{"manage" + g[:4] + g[24:]},
-			Storage: []string{"storage" + g[:4] + g[24:]},
+			Manage:  []string{"manage" + utils.GenUUID()},
+			Storage: []string{"storage" + utils.GenUUID()},
```
Comment (obnoxxx) [inline apps/glusterfs/node_entry_test.go]: you could have just used `g` without truncation here. But I guess it's ok to waste 2 uuids on this. ;-)

**Comment by obnoxxx** [comment]
retest this please

**Comment by obnoxxx** [comment]
@phlogistonjohn needs rebase

**Comment by phlogistonjohn** [comment]
Rebased.

**Comment by obnoxxx** [comment]
retest this please

**Comment by obnoxxx** [comment]
retest this please

**Comment by obnoxxx** [comment]
retest this please

**Comment by nixpanic** [comment]
retest this please - should be fixed, see gluster/glusterfs-patch-acceptance-tests#120

**Comment by nixpanic** [comment]
retest this please

**Comment by obnoxxx** [comment]
retest this please

**Comment by obnoxxx** [comment]
This needs rebase. I can do the "Update branch" button .... @phlogistonjohn

**Comment by phlogistonjohn** [comment]
Rebased. Could have sworn I did it recently but I guess not.

**Comment by obnoxxx** [comment]
@phlogistonjohn but then you merged PR #1004. So this got out of date ;-)

**Comment by obnoxxx** [comment]
no possible effect on the functional tests... moving on

**Pull request merged**